Packs/McAfeeNSM/Integrations/McAfeeNSMv2/McAfeeNSMv2_test.py

```diff
@@ -1083,3 +1088,400 @@ def test_update_alerts_command(args, expected_error, mcafeensmv2_client):
     with pytest.raises(Exception) as e:
         update_alerts_command(mcafeensmv2_client, args)
         assert expected_error == str(e.value)
+
+
+@pytest.mark.parametrize('input, output', [(777, {'method': 'GET', 'url_suffix': '/domain/9/policyassignments/device/777'}),
+                                           (None, {'method': 'GET', 'url_suffix':
+                                                   '/domain/9/policyassignments/device'})])
+def test_list_device_policy_request__with_and_without_device_id(mocker, mcafeensmv2_client, input, output):
+    """
+    Given:
+        - A device id or no device id.
```

Comment:
I prefer to have a description for each test case, I think it is more clear. 
Relevant to all tests that have more than one use case. 
```suggestion
        1. A device id is given. 
        2. The device id isn't given. 
```